_posts/0000-01-02-manojkumar44.md

```diff
@@ -0,0 +1,6 @@
+---
+layout:slide
+title:"Welcome to our second slide!"
+---
+Your text
```

Comment:
**This is my own text which I've added and I'm making it bold**

Furthermore, this is a link to the latest website I've designed and developed:
[Purl Construction Ltd] (https://www.purlconstructionltd.co.uk)